search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -20,16 +20,18 @@
 import io.camunda.search.clients.transformers.ServiceTransformers;
 import io.camunda.search.filter.UserTaskFilter;
 import io.camunda.search.filter.VariableValueFilter;
-import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class UserTaskFilterTransformer implements FilterTransformer<UserTaskFilter> {
 
   private final ServiceTransformers transformers;
+  private final boolean camundaExporterEnabled;
```

Comment:
[Suggestion] naming small suggestion :) 
```suggestion
  private final boolean isCamundaExporterEnabled;
```